app/components/ak-select/index.hbs

```diff
@@ -16,40 +16,77 @@
     {{/if}}
 
     <div
-      local-class='ak-select-trigger-root {{if @label "ak-form-label-spacing"}}'
+      local-class='ak-select-trigger-root 
+      {{if @label "ak-form-label-spacing"}} 
+      {{if @multiple "ak-select-trigger-root-multiple"}}'
     >
-      <PowerSelect
-        @triggerId={{this.labelId}}
-        @dropdownClass='{{@dropdownClass}} {{this.classes.dropdown}}'
-        @triggerClass='
+      {{#if @multiple}}
+        <PowerSelectMultiple
```

Comment:
since all the arguments are same we can simplify this to
```
{{#let (component (if @multiple 'power-select-multiple' 'power-select')) as |SelectComponent|}}
  <SelectComponent as |option|>
   {{yield option}}
  </SelectComponent>
{{/let}}
```